evanxd.md

```diff
@@ -1,41 +1,23 @@
-## 07/11 - 07/15 ##
+## 07/18 - 07/22 ##
 
 ### Last week
 * [Firefox]
-  - Setup development environment
-    - Fixed the try problems: cannot push try
-      - Bug 1286200 - Request hg access be restored
-    - Made a new repo[1] to share documents and config files for new guys
-  - Watch Location Bar module
-    - Bug 647162 - Location Bar does not update
-      - RESOLVED WORKSFORME
   - Dev Tools
-    - Bug 1282791 - Reps: uninteresting props algorithm is wrong
-      - Fixed the uninteresting props issue
-      - Fixed the max number issue
-    - Rep Tester[2]
-      - Be a collaborator of Rep Tester
-      - Issue 1 for prototypes - Add config file to config mozilla-central and devtools paths[3], landed
-      - Issue 1 for rep-tester - Remove Source Directory Structure section[4], landed
-      - Filed new bugs
-        - Issue 2 - Need to run `npm install` twice to install all dependency modules
-        - Issue 3 - Improve the process of running Rep Tester
-
-* [SensorWeb]
-  - Bug 1286142 - Update contributor list
+    - Helped Taipei Dev Tools members understand how to use Rep Tester
+      - Currently Taipei Dev Tools members can use Rep Tester.
+    - Bug 1282791 - Reps: uninteresting props algorithm is wrong, review+
+    - Bug 1276376 - Reps: uninteresting props are not filtered correctly
+      - Fixed this in Bug 1282791
+    - Bug 1284838 - Reps: render events more uniformly, feedback?
 
 ### This week
```

Comment:
Can you drop the last week - this week format? We have not been using this (and rely on commit diff for a long time). Thanks.